backend/app/views/spree/admin/orders/index.html.erb

```diff
@@ -118,6 +118,13 @@
           </div>
         </div>
 
+        <div class="col-xs-12 col-md-4">
+          <div class="form-group">
+            <%= label_tag :q_channel_in, Spree.t(:channel) %>
+            <%= f.select :channel_in, Spree::Order.pluck(:channel).uniq, { include_blank: true }, class: 'select2 js-filterable' %>
```

Comment:
Could you try benchmarking `Spree::Order.pluck('DISTINCT channel')` and `Spree::Order.pluck(:channel).uniq` to determine which one is faster for large amount of records?
I suppose the first version should be better since it's raw SQL but I'm not 100% sure.

Reply:
@dimon-k as @kushniryb suggested - with large amount of Orders this would be painfully slow - let's use SQL `DISTINCT` to limit number of records